hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -144,11 +145,17 @@ spec = do
         update bobEnv ledger s3 (ackFrom aliceSk alice)
           `shouldBe` Error (RequireFailed "requireVerifiedMultisignature")
 
-      it "waits if we receive a snapshot with not-yet-seen transactions" $ do
+      it "waits if we receive a snapshot with not-yet-applicable transactions" $ do
         let event = NetworkEvent defaultTTL $ ReqSn alice 1 [SimpleTx 1 (utxoRef 1) (utxoRef 2)]
         update bobEnv ledger (inOpenState threeParties ledger) event
           `shouldBe` Wait (WaitOnNotApplicableTx (ValidationError "cannot apply transaction"))
 
+      it "waits if we receive a snapshot with not-yet-seen (but applicable) transactions" $ do
+        let s0 = inOpenState threeParties ledger
+            reqSn = NetworkEvent defaultTTL $ ReqSn alice 1 [SimpleTx 1 mempty (utxoRef 1)]
+        update bobEnv ledger s0 reqSn
+          `shouldBe` Wait (WaitOnSeenTx 1)
```

Comment:
This would be exactly the behavior if we would not submit transactions with the `ReqSn`  as actually specified. Do you think this is different in the overall semantics seen by the users?

Reply:
I am sorry, I don't understand your answer